cordova-lib/src/cordova/lazy_load.js

```diff
@@ -284,4 +286,47 @@ function custom(platforms, platform) {
     });
 }
 
+// Returns a promise
+function git_clone(repository) {
```

Comment:
refactored plugins.js and lazy_load.js to use the same code for git cloning and checkout : https://github.com/MSOpenTech/cordova-lib/commit/f7ce4fb55aa55168023bbc62592c6323c81e6b8c